### DISCUSSION TITLE
ipstack.com - allowlist ?

### DISCUSSION THREAD

**Discussion by hasancemcerit** [StevenBlack/hosts#1896] [General]
Hi @StevenBlack, what is the reason of blocklisting `ipstack.com` and `api.ipstack.com` ?
Thank you.

**Comment by StevenBlack** [comment]
Hello Hasan @hasancemcerit thank you for your question.

We're mostly an aggregator of hosts files from active and reputable curators of hosts files, and we package these amalgamated hosts files in various ways.

those `ipstack.com` domains are from upstream.  They come to us via yoyo.org. If you feel `ipstack.com` should not be listed, please make your case to @pgl at pgl@yoyo.org.

**Comment by pgl** [comment]
Hi there. Sorry for the very late reply on this issue - I made some comments elsewhere and didn't follow up here.

The privacy policy for this site explicitly says that they disclose your information to third parties:

> Section 7 of this Privacy Statement, Website Navigational Information, specifically addresses the information we or third parties collect through cookies and web beacons, and how you can control cookies through your Web browser. We may also disclose your personal information to any third party with your prior consent.

So unless I get a request to limit the entries in my list to subdomains, I'm keeping `ipstack.com` on my list, sorry.

**Reply by rautamiekka** [reply]
> with your prior consent

**Comment by ler762** [comment]
On 6/12/22, Jouni Järvinen wrote:
>> with your prior consent

They have a very narrow definition of "your personal information", so
"with your prior consent" is irrelevant to most people.  What does
look applicable to most everyone is the "Service Providers." section:
  ... the Company may also share Data about the Company website
visitors, customers and webinar attendees with the Company’s service
providers to ensure the quality of information provided, and with
third–party social networking and media websites, such as Facebook,
for marketing and advertising on those websites.

**Reply by pgl** [reply]
I mean, yeah, there's lots more in the privacy policy - I only pasted a small quote.

**Comment by ler762** [comment]
On 6/12/22, Peter Lowe wrote:

> I mean, yeah, there's lots more in the privacy policy - I only pasted a
> small quote.

My remark wasn't directed at you - I was responding to the "with your
prior consent" comment that made it look like the site wasn't sharing
your info.
